src/Malt.jl

```diff
@@ -103,19 +103,33 @@ mutable struct Worker <: AbstractWorker
     function Worker(; env=String[], exeflags=[])
         # Spawn process
         cmd = _get_worker_cmd(; env, exeflags)
-        proc = open(Cmd(
-            cmd; 
-            detach=true,
-            windows_hide=true,
-        ), "w+")
+        err = Pipe()
+        proc = open(
+            pipeline(
+                Cmd(
+                    cmd; 
+                    detach=true,
+                    windows_hide=true,
+                ),
+                stderr = err
+            ),
+            "w+"
+        )
         
         # Keep internal list
         __iNtErNaL_get_running_procs()
         push!(__iNtErNaL_running_procs, proc)
 
         # Block until reading the port number of the process (from its stdout)
         port_str = readline(proc)
-        port = parse(UInt16, port_str)
+        port = tryparse(UInt16, port_str)
+        if port === nothing
+            Base.kill(proc, Base.SIGTERM)
+            close(err.in)
+            err_t = Threads.@spawn read(err, String)
+            err_output = fetch(err_t)
```

Comment:
Why not just `err_output = read(err, String)`? Can you add a comment?

Reply:
Good question, I think this was an artifact of me moving things around. It doesn't make sense this way anymore.